### PR TITLE
Added IIB override option to Internal installation

### PR DESCRIPTION
**Pull request by nathan-weinberg**
Based on Slack huddle

### CODE REVIEW

**Comment by nathan-weinberg** [comment]
Test using `Internal` with `IIB_OVERRIDE`: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/53/
Test using `Internal` without `IIB_OVERRIDE`: https://mastern-jenkins-csb-openshift-qe.apps.ocp-c1.prod.psi.redhat.com/job/scale-ci/job/nweinber-e2e-benchmarking-multibranch-pipeline/job/netobserv-perf-tests/54/

File: scripts/netobserv.sh [hunk 1]
```diff
@@ -80,7 +80,7 @@ deploy_lokistack() {
   if [[ $WORKLOAD == "None" ]] || [[ -z $WORKLOAD ]]; then
     export S3_BUCKETNAME="netobserv-ocpqe-perf-loki-$RAND_SUFFIX"
   else
-    export S3_BUCKETNAME="netobserv-ocpqe-perf-loki-$WORKLOAD"
+    export S3_BUCKETNAME="netobserv-ocpqe-perf-loki-$WORKLOAD-$RAND_SUFFIX"
```
Comment (memodi) [inline scripts/netobserv.sh:83]: I guess, it's make sense if those old buckets does exist for runs made in succession, with this we'd be sure to use the new bucket every time - was that the reason to change this or there's more to it?

File: scripts/netobserv.sh [hunk 2]
```diff
@@ -124,6 +124,11 @@ deploy_unreleased_catalogsource() {
   oc apply -f $SCRIPTS_DIR/icsp.yaml
 
   echo "====> Determining CatalogSource config"
+  if [[ -z $IMAGE ]]; then
```
Comment (memodi) [inline scripts/netobserv.sh:127]: Could be additional log with `echo` here to specify which image it ends up using in build log? 

I know we don't mention about installing from "Internal" on our Readme, just Source and OperatorHub. Do you think we should ? 
 if we do, we should also mention about setting `IMAGE` ,  `MAJOR_VERSION` and `MINOR_VERSION` .

**Review by memodi — CHANGES_REQUESTED** [review]
looks good overall, just a minor comment/question.

File: scripts/catalogsources/qe-unreleased-catalogsource.yaml [hunk 3]
```diff
@@ -5,5 +5,5 @@ metadata:
   namespace: openshift-marketplace
 spec:
   displayName: NetObservUnreleasedTesting
-  image: 'quay.io/openshift-qe-optional-operators/aosqe-index:v${MAJOR_VERSION}.${MINOR_VERSION}'
+  image: ${IMAGE}
```
Comment (Amoghrd) [inline scripts/catalogsources/qe-unreleased-catalogsource.yaml:8]: This is what I was asking about, I like this approach setting values in YAML files

File: scripts/netobserv.sh (hunk 1 above)
Comment (nathan-weinberg) [reply scripts/netobserv.sh:83]: Honestly, the reason behind it was that I've seen collisions where more than one person was running the same workload or cases when a bucket wasn't cleaned up and there were errors due to the bucket already exisiting

File: scripts/netobserv.sh (hunk 2 above)
Comment (nathan-weinberg) [reply scripts/netobserv.sh:127]: I will add an `echo` and update the `README`!

File: scripts/catalogsources/qe-unreleased-catalogsource.yaml (hunk 3 above)
Comment (nathan-weinberg) [reply scripts/catalogsources/qe-unreleased-catalogsource.yaml:8]: Yes I am going to see if there is additional consolidation we can do in future PRs!

**Pull request merged**